doc/2/api/protocols/http/index.md

```diff
@@ -64,3 +64,22 @@ The listening port can be modified under the `server.port` section of the [confi
 HTTP and WebSocket protocols share the same underlying server instance.  
 Modifying the listening port will impact these two protocols.
 :::
+
+### HTTP-only options
+
+Only the HTTP protocol does support the following options in a [KuzzleRequest](../../payloads/request/index.md)
```

Comment:
We usually use absolute paths in URLs. Also, `index.md` won't exist once the documentation have been generated, since markdowns are used to build HTML files.

When unsure, don't hesitate to test your documentation by running `npm run doc-dev` and by navigating the local copy of the docs on your browser.

```suggestion
Only the HTTP protocol does support the following options in a [KuzzleRequest](/core/2/api/payloads/request)
```